src/app/Model/menu-item.ts

```diff
@@ -0,0 +1,26 @@
+import { Httpable } from "../httpable";
+
+export class MenuItem extends Httpable  
+{
+    name:string = '';
+    description:string = '';
+    image:string = '';
+    constructor(name:string, description:string, image:string){
+        super();
+        this.name = name;
+        this.description = description;
+        this.image = image;
+    }
+
+    public setDescription(description:string){
```

Comment:
Typescript has special ways of making get and set methods. Don't worry about changing these though